lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2587,6 +2592,11 @@ LogicalResult StmtEmitter::visitStmt(OutputOp op) {
   return success();
 }
 
+LogicalResult StmtEmitter::visitStmt(TypeScopeOp op) {
+  TypeScopeEmitter(*this).emitTypeScopeBlock(*op.getBodyBlock());
```

Comment:
See my other comment. I think right here we could just say `StatementEmitter(...).emitStatementBlock(op.getBodyBlock)`. And we would need to add the visitor and `visitStmt` for TypeDeclOp.